manim/mobject/svg/code_mobject.py

```diff
@@ -370,22 +484,27 @@ def hilite_me(code, lexer, options, style, linenos, divstyles):
         cssstyles=defstyles + divstyles,
         prestyles="margin: 0",
     )
-    html = highlight(code, get_lexer_by_name(lexer, **options), formatter)
-    if linenos:
+    if language is None:
+        lexer = guess_lexer_for_filename(file_path, code)
+        html = highlight(code, lexer, formatter)
+    else:
+        html = highlight(code, get_lexer_by_name(language, **{}), formatter)
+    if insert_line_no:
         html = insert_line_numbers(html)
-    html = "<!-- HTML generated using hilite.me -->" + html
+    html = "<!-- HTML generated by Code() -->" + html
     return html
 
 
-def get_default_style():
-    return "border:solid gray;border-width:.1em .1em .1em .8em;padding:.2em .6em;"
-
-
 def insert_line_numbers(html):
+    """Function put line numbers to html of highlighted code.
+    Parameters
```

Comment:
Line between description and section
```suggestion

    Parameters
```